src/sentry/utils/audit.py

```diff
@@ -42,10 +42,19 @@ def create_audit_entry_from_user(
     ip_address: str | None = None,
     transaction_id: int | str | None = None,
     logger: Logger | None = None,
+    organization: Organization | None = None,
+    organization_id: int | None = None,
     **kwargs: Any,
 ) -> AuditLogEntry:
+    if organization:
+        organization_id = organization.id
```

Comment:
Do we want to guard against the scenario where both `organization` and `organization_id` are provided at the same time?

Reply:
Added in latest commit